valve-monitor/web-app/src/pages/api/device/[deviceUID]/name.ts

```diff
@@ -0,0 +1,78 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```

Comment:
I'd remove this file and let `/api/device/[deviceUID]/valve-monitor-config.ts` handle all device update logic

Reply:
It definitely ended up being a lot cleaner 🙂 6264004e5ae179b79469b8431f47d2ac4e554fe5

Thanks!